### PR TITLE
test comm

### PR DESCRIPTION
**Pull request by smarterhicomm**
test code inspection

### CODE REVIEW

**Review by amey-smarterhi — APPROVED** [review]
Approved

File: TestFile.java [hunk 1]
```diff
@@ -14,6 +14,8 @@
 public class TestFile {
 	private Connection conn;
 
+	// Test my changes 
+	
 	public static void main(String[] args) {
 		// TODO Auto-generated method stub
 		String test = "[7]";
```
Comment (preeti-smarterhi) [inline TestFile.java:21]: Testing review comment

Comment (smarterhicomm) [reply TestFile.java:21]: Ok will do that.

**Review by preeti-smarterhi — CHANGES_REQUESTED** [review]
PLease make changes

**Pull request merged**